Models/c172p.xml

```diff
@@ -2539,6 +2540,10 @@
                 <command>property-toggle</command>
                 <property>controls/anti-ice/engine[0]/carb-heat</property>
             </binding>
+            <binding>
```

Comment:
I would use `/controls/engines/current-engine/carb-heat` via an alias here (and on 2524) and add the property to `<params><controls>` somewhere at line 258. This prepares the aircraft for having controls transmitted over MP (I already did the same preparation for throttle and mixture)

Also add two filters that write an output to `/controls/anti-ice/engine[0]/carb-heat` and `/controls/anti-ice/engine[1]/carb-heat`. You can basically take the two filters for mixture (line 366 to 404), modify them, and insert the somewhere on line 405. Plus add a little comment block:
```
<!-- ============================================================== -->
<!-- Carburator Heat                                                -->
<!-- ============================================================== -->
```

You can replace `<!-- Fuel Contamination and low oil level-->` with a block like above as well. I use the three line comments to visually group the filters.